### DISCUSSION TITLE
hackintosh/updates/2022/08/02/acidanthera-dortania-august

### DISCUSSION THREAD

**Discussion by giscus** [dortania/dortania.github.io#34] [Comments]
# hackintosh/updates/2022/08/02/acidanthera-dortania-august

Acidanthera Updates: A message from PMheart Dortania Updates: A message from dhinakg Changelogs

https://dortania.github.io/hackintosh/updates/2022/08/02/acidanthera-dortania-august.html

**Comment by perez987** [comment]
We have missed these messages for a long time! Thank you for coming back. I like them very much. Thank you also for your work, I know all of you are busy sometimes and in spite of everything things go on and are good.
I think it's a good idea the change of the guides from CPU generation to config.plist sections. Not mandatory for me as user since all the info is well organized there but, as you say, maybe the new way makes things easier for the writers.

**Comment by NZRichard** [comment]
Yes, organizing web content for the Install Guide so it's easier to update and maintain is important, and it can also make your web traffic more efficient and reduce the load on your server.

The reference to "cutting and pasting" sounds like the guide is maybe a series of web pages that are downloaded to a browser when the user makes a request by clicking a link in the guide.

Some web designers will instead retrieve a few lines or a paragraph of text or a graphic to replace or add to content that is already in the browser window. This is much more efficient than downloading or refreshing the entire page of content.

There are a few different ways to call and display content without reloading the entire page. You may have come across Ajax which uses XMLHttpRequest (XHR) objects  as explained here: https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest

Sounds like some good opportunities here!

**Reply by HippoInWindow20** [reply]
The site is powered by GitHub pages, so the retrieval method isn’t really up to the content creator. Instead documentation is created with Markdown as individual files. Unless all pages have to be rewritten with HTML or a different framework, it seems unlikely

**Reply by HippoInWindow20** [reply]
In addition, the framework only loads the menu bar and other necessary features upon first load, switching between pages is essentially loading different markdown files

**Reply by HippoInWindow20** [reply]
Finally, since the website is hosted by GitHub, server load shouldn’t really be an issue ;)

**Reply by averycblack** [reply]
We currently use Vuepress to compile markdown into webpages which are served by GitHub pages.

**Comment by nuche** [comment]
The news about consolidation and reduction of maintenance workload is wonderful to hear. I think homogeneous presentations that are cleaner and easier to update are really really good and easier to contribute to.

@Goldfish64 has been doing some really interesting work on [Hyper-v support](https://github.com/acidanthera/MacHyperVSupport). Could we get a guide section dedicated to Hyper-v installation to what we may contribute and work with to install macOS in Hyper-v servers and such? It seems there must be ways to get it working but it seems people trying to get it to work are struggling.

**Comment by hh724330184** [comment]
Great job！！！！！！！

**Comment by yaduramauk** [comment]
Please Include Alderlake and Tiger Lake (Experimental) Configs> Intel Desktop config.plist in OpenCore Install Guide

**Comment by puptruk** [comment]
I have been a hackintosher for a brief 6 years. I started out leaning and using other means to accomplish my builds. I genuinely appreciate everyone on your team's dedication and hard work. If I had more knowledge and skill at your craft, I would gladly help with these project. Thank you so very, very much for your work. I GREATLY value Opencore!!

**Comment by pmdevita** [comment]
I think as long as the organization makes sense (and I think it does), the guide changes will be fine. Since it isn't step by step tailored for a CPU generation anymore, it might be good to provide some kind of introduction and explain to users how to read and use the guide.

**Comment by thefat32** [comment]
I think the idea of creating better organized pages based in OpenCore configuration sections instead of proccessor generation is great. It will reduce the duplicated info and allow to maintain generation specific configurations/quirks better

**Comment by Danish297** [comment]
I see aug'22 update bit late, but love the work and changes you ppl made. initially, I tried Hackintosh in early 2009 and the only option is for me to "boot with CD" and use it. Later I brought my first MacBook pro in 2011. But in late 2020, OC make my dream alive to use the most advoked macOS out there, and it still surprises me "OMG, I can do this, in this way too" then I build my first Hackintosh with OC in late 2020. Thank you guys once again 4 great work.
